go/oasis-node/cmd/node/node_control.go

```diff
@@ -343,6 +343,17 @@ func (n *Node) getRuntimeStatus(ctx context.Context) (map[common.Namespace]contr
 			}
 		}
 
+		// Fetch last consensus height seen by block history.
+		height, err := rt.History().LastConsensusHeight()
+		if err == nil {
+			n.logger.Error("failed to retrieve last consensus height seen by block history",
+				"err", err,
+				"id", rt.ID(),
+			)
+		} else {
```

Comment:
[nit] I'm not a big fan of `if/else`, as it makes code less readable and the reader normally expects only `if` part which handles the error. To be in-line with the code above, you could assign directly to the status field (even if there is an error, assuming that the returned value is zero value) or use `switch` like it is done above. I would use the former, just to be in-line with code above.

Maybe this code could be moved up, where we handle history.